### PR TITLE
Structural changes and introduction edits

### PR DESCRIPTION
**Pull request by cgreene**
I'm in the process of reading through the paper and making revisions. I will aim to complete the read-through this week. I made a few structural changes to set things up for a smoother review process, including moving some details from the intro to methods.

### CODE REVIEW

File: content/05.main-text.md [hunk 1]
```diff
@@ -91,25 +56,14 @@ but requires supervision in the form of known treatments.
 <!-- info on this figure's creation in https://github.com/greenelab/connectivity-search-manuscript/issues/11 -->
 ](https://github.com/greenelab/connectivity-search-manuscript/raw/f98c3470a8bf8f40f5f6aed2794c6ea66b93b14b/content/media/rephetio/metagraph-and-features.png){#fig:rephetio width="100%" .white}
 
-### Unsupervised hetnet connectivity search
-
-Project Rephetio was able to successfully predict treatments,
+Project Rephetio was successfully predict treatments,
```
Comment (dhimmel) [inline content/05.main-text.md]:
```suggestion
Project Rephetio successfully predicted treatments,
```

File: content/05.main-text.md [hunk 2]
```diff
@@ -922,6 +913,14 @@ We encourage readers with feedback or questions to comment publicly via [GitHub
 
 ### Software & data availability
 
+#### Hetio
+
+*Hetio* is a superset/collection of hetnet-related research, tools, and datasets that, most notably, includes the Hetionet project itself and the connectivity search tool that are the focus of this manuscript.
+Most of the Hetio resources and projects can be found under the [Hetio GitHub organization](https://github.com/hetio), with a others being available under the [Greene Lab GitHub organization](https://github.com/greenelab), one of the collaborating groups.
+Information about Hetio is also displayed and disseminated on the [Hetio website](https://het.io/), as discussed in more detail below.
```
Comment (dhimmel) [inline content/05.main-text.md]:
```suggestion
Information about Hetio is also displayed and disseminated on the [Hetio website](https://het.io/), as noted in the [Hetio Website] section. 
```

**Review by dhimmel — COMMENTED** [review]
Title of PR title says DRAFT, but I think I should merge now after committing these suggestions since there are some text relocations and frequent merges will help avoid conflicts.

**Comment by cgreene** [comment]
That works for me - I had hoped to complete this week but didn't. Feel free to merge and I can update to main and then branch off again from there. Thanks!

**Pull request merged**